spec.emu

```diff
@@ -290,15 +1188,8 @@ contributors: Luca Casonato, Guy Bedford, Nicolò Ribaudo
                 <ins>an Object or *null* (defaults to *null*)</ins>
               </td>
               <td>
-                <ins>An Object corresponding to the source Module Record.</ins>
-
-                <emu-note type="editor">
-                  <p>This concept is better explained by <a href="https://tc39.es/proposal-compartments/0-module-and-module-source.html">Layer 0</a> and <a href="https://github.com/tc39/proposal-compartments/blob/master/2-virtual-module-source.md">Layer 2</a> of the Compartments proposal.</p>
-
-                  <p>The *null* default will throw an error for unsupported phaseions (including for SourceTextModuleRecord).</p>
-
-                  <p>For <a href="https://webassembly.github.io/esm-integration/js-api/index.html#esm-integration">WebAssembly modules</a>, this field should contain the corresponding `WebAssembly.Module` object.</p>
-                </emu-note>
+                <ins>The ModuleSource Object corresponding to this source Module Record's source phase (<emu-xref href="#sec-module-source-objects"></emu-xref>), if it has been requested.</ins>
+                <ins>This object must have a [[ModuleSourceRecord]] internal slot and %AbstractModuleSourcePrototype% at the root of its prototype chain.</ins>
```

Comment:
```suggestion
                <ins>This object must have a [[ModuleSourceRecord]] internal slot and %AbstractModuleSourcePrototype% in its prototype chain.</ins>
```

`%AbstractModuleSourcePrototype%` has `Object.prototype` as it's [[Prototype]], so it's not the root :P